Grid/grid.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Grids</title>
+
+    <link rel="stylesheet" href="/styles.css">
+    <link rel="stylesheet" href="/Grid/grid.css">
+</head>
+
+<body>
+    <header class="home-header flex">
+        <img class="logo" src="/images/comp-lib-logo.png" alt="Brand-logo">
+        <h1 class="brand">Garnish UI</h1>
+        <ul class="nav-links flex">
+            <li><a class="nav-link" href="/index.html">Home</a></li>
+            <li><a class="nav-link link-active" href="/documentation.html">Documentation</a></li>
+        </ul>
+    </header>
+
+    <main class="docs-main">
```

Comment:
you've added side bar in the main, so main tag is used to write the HTML content which is unique & not repetitive in nature. Since side bar is repitive in every page, you can use only aside tag